src/main/java/edu/kis/vh/nursery/factory/ListRhymersFactory.java

```diff
@@ -3,23 +3,28 @@
 import edu.kis.vh.nursery.DefaultCountingOutRhymer;
 import edu.kis.vh.nursery.FIFORhymer;
 import edu.kis.vh.nursery.HanoiRhymer;
-import edu.kis.vh.nursery.dataStructures.IntArrayStack;
+import edu.kis.vh.nursery.dataStructures.IntLinkedList;
 
 public class ListRhymersFactory implements Rhymersfactory{
+    IntLinkedList stack;
+
+    public ListRhymersFactory() {
+        stack = new IntLinkedList();
+    }
+
     @Override
     public DefaultCountingOutRhymer getStandardRhymer() {
         return new DefaultCountingOutRhymer();
     }
 
     @Override
     public DefaultCountingOutRhymer getFalseRhymer() {
-        IntArrayStack stack = new IntArrayStack();
         return new DefaultCountingOutRhymer(stack);
```

Comment:
"FalseRhymer" should use IntArrrayStack ("getFalseStack powinna zwrócić stos oparty na implementacji „przeciwnej” do domyślnej.")